.npmignore

```diff
@@ -0,0 +1,28 @@
+dist/stats.json
```

Comment:
also, this doesn't have a big impact in the end, it's just to be nice and put only what is needed in the npm package... if we forget to ignore stuff later, it won't break anything as long as `dist/smooch.js` is still there.